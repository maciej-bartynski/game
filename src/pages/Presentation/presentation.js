import React from 'react';
import { View, Text } from 'react-native';

const style = { flex: 1, justifyContent: 'center', alignItems: 'center' };
export default function Presentation() {
  return (
    <View style={style}>
      <Text>Presentation page</Text>
    </View>
  );
}
