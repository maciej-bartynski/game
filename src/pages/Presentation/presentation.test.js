import React from 'react';
import renderer from 'react-test-renderer';
import Presentation from './index';

describe('<Presentation />', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Presentation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
