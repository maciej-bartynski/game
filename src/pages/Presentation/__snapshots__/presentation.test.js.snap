// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<Presentation /> matches snapshot 1'] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Presentation page
  </Text>
</View>
`;
